### PR TITLE
Fix/xxx  template mail fix

### PR DESCRIPTION
**Pull request by inkhey**
Few fix on templates

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

### CODE REVIEW

**Comment by inkhey** [comment]
in draft, because in wait for potential others change in template mail.

**Comment by lebouquetin** [comment]
As suggested this morning when talking to @PhilippeAccorsi we should replace all wording like " XXX allowed you to upload files" by something like : "XXX invited you to upload files".

By the way, in French, the translations should be "Envoyer vos fichiers" ou "Téléversez". Le terme d'import / importation correspond plutôt à un cas où "j'importe mes fichiers dans mon espace", hors là il s'agit de personnes extérieures à tracim (en ce sens elle n'importent pas mais elles envoient - téléverses - import a une notion de "amener chez soi")

**Comment by inkhey** [comment]
> As suggested this morning when talking to @PhilippeAccorsi we should replace all wording like " XXX allowed you to upload files" by something like : "XXX invited you to upload files".
> 

ok. it's ok for subject but removing "allow" in template itself is a bit more complicated:
we got : 
- "I allowed you to upload files here:"
- "You allowed the upload in this shared space <a href={sharedspace_url}>{sharedspace_name}</a> with:"

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 1]
```diff
@@ -59,28 +59,24 @@ msgstr ""
...
-#: tracim_backend/applications/upload_permissions/config.py:23
+#: tracim_backend/applications/upload_permissions/config.py:21
 msgid ""
-"[{website_title}] You allowed {nb_receivers} users to upload files on "
+"[{website_title}] You invited {nb_receivers} people to upload files on "
 "\"{workspace_name}\""
 msgstr ""
-"[{website_title}] Vous avez autorisé {nb_receivers} utilisateurs à "
-"téléverser des fichiers dans \"{workspace_name}\""
+"[{website_title}] Vous avez invité {nb_receivers} personnes à téléverser "
```
Comment (inkhey) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]:
```suggestion
"[{website_title}] Vous avez invité {nb_receivers} personne(s) à téléverser "
```

Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: I suggest this to handle the share to one people case.

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 2]
```diff
@@ -59,28 +59,24 @@ msgstr ""
...
 
 #: tracim_backend/applications/upload_permissions/config.py:14
-msgid ""
-"[{website_title}] {emitter_name} allowed you to upload files in this "
-"shared space \"{workspace_number}\""
-msgstr ""
-"[{website_title}] {emitter_name} vous a autorisé à téléverser des "
-"fichiers dans l'espace partagé \"{workspace_number}\""
+msgid "{emitter_name} invited you to upload files on \"{website_title}\""
+msgstr "{emitter_name} vous a invité à téléverser fichiers sur \"{website_title}\""
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: Missing word I think.

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 1 above)
Comment (PhilippeAccorsi) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: In this case, check everywhare to add `(s)` when it's necessary.

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 3]
```diff
@@ -303,20 +299,16 @@ msgid "Hello,"
...
 
 #: tracim_backend/templates/mail/shared_content_to_receiver_body_html.mak:6
-msgid "I share a file with you."
-msgstr "J'ai partagé un fichier avec vous."
-
-#: tracim_backend/templates/mail/shared_content_to_receiver_body_html.mak:7
-msgid "You can download this file here:"
-msgstr "Vous pouvez télécharger ce fichier ici :"
+msgid "I share a file with you, download it here:"
+msgstr "J'ai partagé un fichier avec vous, télécharger le ici:"
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: Missing space before `:`

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 4]
```diff
@@ -361,15 +353,11 @@ msgstr ""
...
 
 #: tracim_backend/templates/mail/upload_permission_to_receiver_body_html.mak:6
-msgid ""
-"I allowed you to upload file in this shared space (number: "
-"{workspace_id}):"
-msgstr ""
-"Je vous ai autorisé à téléverser des fichiers dans cet espace partagé "
-"(numéro {workspace_id})"
+msgid "I invited you to upload files here:"
+msgstr "Je vous ai invité à téléverser des fichiers ici:"
```
Comment (PhilippeAccorsi) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: missing space before `:`

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 1 above)
Comment (buxx) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: agreed

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 3 above)
[the remark this answers is not in the document]
Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: done.

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 5]
```diff
@@ -59,28 +59,24 @@ msgstr ""
 #: tracim_backend/applications/share/config.py:22
 msgid ""
 "[{website_title}] You shared \"{content_filename}\" with {nb_receivers} "
-"users"
+"people"
 msgstr ""
 "[{website_title}] Vous avez partagé  \"{content_filename}\" avec "
-"{nb_receivers} utilisateurs"
+"{nb_receivers} personnes"
```
[the remark this answers is not in the document]
Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: done.

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 2 above)
[the remark this answers is not in the document]
Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: done.

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 4 above)
Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: done.

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 3 above)
Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: done.

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 2 above)
Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: done.

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po (hunk 1 above)
Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: done.

Comment (inkhey) [reply backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po]: done.

**Pull request merged**